styles.css

```diff
@@ -0,0 +1,127 @@
+*{
```

Comment:
great idea using a global stylesheet to maintain a universal design system throughout your site!